extensions/http-curl/tests/HTTPHandlers.h

```diff
@@ -792,20 +792,40 @@ class C2AcknowledgeHandler : public ServerAwareHandler {
     rapidjson::Document root;
     root.Parse(req.data(), req.size());
     if (root.IsObject() && root.HasMember("operationId")) {
-      std::lock_guard<std::mutex> guard(mtx_);
+      std::lock_guard<std::mutex> guard(ack_operations_mtx_);
       acknowledged_operations_.insert(root["operationId"].GetString());
     }
+
+    if (root.IsObject() && root.HasMember("operationState")) {
+      if (root["operationState"].IsObject() && root["operationState"].HasMember("state")) {
+        std::lock_guard<std::mutex> guard(apply_count_mtx_);
+        auto result_state = root["operationState"]["state"].GetString();
+        if (apply_count_.find(result_state) != apply_count_.end()) {
```

Comment:
note: `++apply_count[result_state]` would achieve the same result

Reply:
Good call! Updated in 4198ad6599cd8ae739342b5376347c60329d53b6